ztag/stream.py

```diff
@@ -176,17 +197,31 @@ def __init__(self, logger=None, destination=None, *args, **kwargs):
         else:
             raise Exception("invalid destination: %s" % destination)
         host = os.environ.get('KAFKA_BOOTSTRAP_HOST', 'localhost:9092')
+        processes = os.environ.get('ZTAG_KAFKA_PROCESSES', 2)
         self.main_producer = KafkaProducer(bootstrap_servers=host)
         self.cert_producer = KafkaProducer(bootstrap_servers=host)
+        self.queue = multiprocessing.Queue()
+        self.processes = list()
+
+        for pnum in xrange(0, int(processes)):
+            p = _KafkaWorkerProcess(
+                    self.queue, self.topic, self.main_producer,
+                    self.cert_producer
+            )
+            p.start()
+            self.processes.append(p)
 
     def take(self, pbout):
-        for certificate in pbout.certificates:
-            self.cert_producer.send("certificate", certificate)
-        self.main_producer.send(self.topic, pbout.transformed)
+        self.queue.put(pbout)
 
     def cleanup(self):
+        # Trigger end of the queue
+        for p in self.processes:
+            self.queue.put(None)
+        # God forbid Python have sane abstractions
```

Comment:
This comment adds no information right. Either explicitly describe the problem or remove the comment.